.idea/inspectionProfiles/Gradle.xml

```diff
@@ -1,31 +1,12 @@
 <component name="InspectionProjectProfileManager">
   <profile version="1.0">
     <option name="myName" value="Gradle" />
-    <inspection_tool class="Anonymous2MethodRef" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="AnonymousHasLambdaAlternative" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="CodeBlock2Expr" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="ComparatorCombinators" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="Convert2Diamond" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="Convert2Lambda" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="Convert2MethodRef" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="Convert2streamapi" enabled="false" level="INFORMATION" enabled_by_default="false" />
-    <inspection_tool class="EqualsReplaceableByObjectsCall" enabled="false" level="WEAK WARNING" enabled_by_default="false" />
-    <inspection_tool class="FoldExpressionIntoStream" enabled="false" level="INFORMATION" enabled_by_default="false" />
-    <inspection_tool class="Guava" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="Java8ArraySetAll" enabled="false" level="INFORMATION" enabled_by_default="false" />
-    <inspection_tool class="Java8CollectionRemoveIf" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="Java8ListSort" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="Java8MapApi" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="Java8MapForEach" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="LambdaCanBeMethodCall" enabled="false" level="INFORMATION" enabled_by_default="false" />
-    <inspection_tool class="SafeVarargsDetector" enabled="false" level="WARNING" enabled_by_default="false" />
-    <inspection_tool class="SimplifyForEach" enabled="false" level="INFORMATION" enabled_by_default="false" />
-    <inspection_tool class="Since15" enabled="true" level="ERROR" enabled_by_default="true">
-      <effectiveLL value="JDK_1_8" />
-    </inspection_tool>
```

Comment:
I don't think we should remove this one. Our code still needs to run on JDK8.

Reply:
This is still enabled, we're just not hard-coding the settings to Java 8, rather that it should take into consideration the language level of the module:
<img width="382" alt="Screen Shot 2019-07-15 at 22 49 51" src="https://user-images.githubusercontent.com/1841944/61248240-f456d900-a752-11e9-8ab8-c1732938ccbf.png">

Reply:
I've opened https://youtrack.jetbrains.com/issue/IDEA-218172 for Groovy & Kotlin as well

Reply:
I don't think this would work. Currently every project is set to language level 11 in my workspace with the default settings. We need to fix this separately (there are some cyclic dependencies that need to be fixed before IIRC?).

Reply:
For the modules, yes:
<img width="254" alt="Screen Shot 2019-07-16 at 10 31 21" src="https://user-images.githubusercontent.com/1841944/61278656-e174f080-a7b4-11e9-824a-a798199cdc61.png">

but the individual source folders seem to be correctly set to 8 or 6 - seems to me:
<img width="355" alt="Screen Shot 2019-07-16 at 10 29 43" src="https://user-images.githubusercontent.com/1841944/61278575-b68a9c80-a7b4-11e9-86b4-b12a958ebc2e.png">
<img width="304" alt="Screen Shot 2019-07-16 at 10 29 49" src="https://user-images.githubusercontent.com/1841944/61278576-b7233300-a7b4-11e9-9312-17f8a331dcc5.png">

Reply:
You are right. In that case I agree it's a good idea to remove this.